lib/swagger/generator.rb

```diff
@@ -28,8 +43,8 @@ def resource_name
       @resource_name ||= collection_name.singularize
     end
 
-    def generate_swagger_schemas
-      name = resource_name
+    def generate_swagger_schemas(name=false)
```

Comment:
@DenisPopov15 yes, sorry. You are right, I've messed this up with CoffeeScript. Still, please change default `false` value to default `nil`.